python/test/t_Gibbs_std.py

```diff
@@ -1,14 +1,66 @@
 #! /usr/bin/env python
-
+# %%
```

Comment:
While useful during the coding phase, these # %% can be removed from the final file.